packages/tempus/test/DIRK/Tempus_DIRKTest.cpp

```diff
@@ -578,8 +578,8 @@ TEUCHOS_UNIT_TEST(DIRK, EmbeddedVanDerPol)
          //getCurrentState()->getMetaData()->getNFailures();
          const int iStep = integrator->getSolutionHistory()->
             getCurrentState()->getIndex();
-         const int nFail = integrator->getSolutionHistory()->
-            getCurrentState()->getMetaData()->getNRunningFailures();
+         //const int nFail = integrator->getSolutionHistory()->
+         //   getCurrentState()->getMetaData()->getNRunningFailures();
```

Comment:
Did you mean to keep this commented out code?

Reply:
Yeah, nFail is never used.  We could remove it.